test/extractor/AssignmentRuleExtractor.test.ts

```diff
@@ -179,17 +179,13 @@ describe('AssignmentRuleExtractor', () => {
       element.patternQuantity.system = 'http://other-units.org';
       element.patternQuantity.unit = 'Gigawatts';
       const assignmentRules = AssignmentRuleExtractor.process(element);
-      const expectedQuantityRule = new ExportableAssignmentRule('valueQuantity');
-      expectedQuantityRule.value = new fshtypes.FshCode(
-        'GW',
-        'http://other-units.org',
-        'Gigawatts'
+      const expectedRule = new ExportableAssignmentRule('valueQuantity');
+      expectedRule.value = new fshtypes.FshQuantity(
+        1.21,
+        new fshtypes.FshCode('GW', 'http://other-units.org', 'Gigawatts')
       );
-      const expectedValueRule = new ExportableAssignmentRule('valueQuantity.value');
-      expectedValueRule.value = 1.21;
-      expect(assignmentRules).toHaveLength(2);
-      expect(assignmentRules[0]).toEqual<ExportableAssignmentRule>(expectedQuantityRule);
-      expect(assignmentRules[1]).toEqual<ExportableAssignmentRule>(expectedValueRule);
+      expect(assignmentRules).toHaveLength(1);
+      expect(assignmentRules[0]).toEqual<ExportableAssignmentRule>(expectedRule);
```

Comment:
Since we now don't special case UCUM, I don't think we need this test at all anymore, since it just tests the same path of code as the test above.